siphon/catalog.py

```diff
@@ -454,6 +454,12 @@ def __init__(self, element_node, catalog_url=''):
 
         """
         self.name = element_node.attrib['name']
+
+        if 'ID' in element_node.attrib:
+            self.id = element_node.attrib['ID']
```

Comment:
`self.id = element_node.attrib.get('ID', None)` would be shorter... 😁

Reply:
Could technically do that to the `'urlPath'` block below if you're feeling motivated...